src/rgen/Microsoft.Macios.Transformer/DataModel/DelegateParameter.Transformer.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis;
+using Microsoft.Macios.Transformer.Attributes;
+
+namespace Microsoft.Macios.Generator.DataModel;
+
+readonly partial struct DelegateParameter {
```

Comment:
isn't this a dupe file ?

Reply:
It is not, because the definition of the attribute is different :) 

The transformer and the generator reference to two different attribute definitions. It is a little trick I am doing not to have to use crazy #if preprocessing in the code.